framework/k4SimDelphes/src/k4SimDelphesAlg.cpp

```diff
@@ -17,10 +17,10 @@ k4SimDelphesAlg::k4SimDelphesAlg(const std::string& name, ISvcLocator* svcLoc)
 
 StatusCode k4SimDelphesAlg::initialize() {
   ///-- setup Configuration and input arrays //////////////////////////////////
-  m_Delphes    = std::make_unique<Delphes>("Delphes");
-  m_confReader = new ExRootConfReader();
-  m_confReader->ReadFile(m_DelphesCard.value().c_str());
-  m_Delphes->SetConfReader(m_confReader);
+  m_Delphes       = std::make_unique<Delphes>("Delphes");
+  auto confReader = ExRootConfReader();
+  confReader.ReadFile(m_DelphesCard.value().c_str());
```

Comment:
This creates a dangling pointer inside Delphes. It doesn't seem to have any effect in the end, but it should still be fixed.